Source/Events.Store.MongoDB/Processing/Streams/EventHorizon/SubscriptionStates.cs

```diff
@@ -49,7 +49,6 @@ public async Task<IMongoCollection<MongoDB.Processing.Streams.EventHorizon.Subsc
                         .Ascending(_ => _.ConsumerTenantId)
```

Comment:
Why do we keep the consumer TenantId? Shouldn't that also be implicit from what event store database this is stored in?

Reply:
Good point, also removed `ConsumerTenantId` from SubscriptionState.